python/semantic_kernel/contents/annotation_content.py

```diff
@@ -47,11 +48,13 @@ class AnnotationContent(KernelContent):
 
     def __str__(self) -> str:
         """Return the string representation of the annotation content."""
-        return f"AnnotationContent(file_id={self.file_id}, url={self.url}, quote={self.quote}, start_index={self.start_index}, end_index={self.end_index})"  # noqa: E501
+        return f"AnnotationContent(type={self.citation_type}, file_id={self.file_id}, url={self.url}, quote={self.quote}, start_index={self.start_index}, end_index={self.end_index})"  # noqa: E501
```

Comment:
won't this output something like `"{"AnnotationContent(type=CitationType.FILE_PATH"` instead of `type=file_path`?